packages/sockethub/package.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Did you intentionally switch to tabs for this one? Seems to be inconsistent with the rest.

Reply:
Oops, not sure how that happened.